feature-group-definitions/web-animations.yml

```
@@ -1,4 +1,5 @@
 name: Web animations
+description: The Web Animations API allows to animate, and synchronize the animations of DOM elements.
```

This API has multiple entry points but I think it's fair to call the `animate()` method the main one. However, I really don't think the feature would be generally recognized by the name "animate", so I guess I'll have to suggest this:

```suggestion
name: Web animations
description: The web animations API allows you to animate and synchronize the animations of DOM elements.
```